### DISCUSSION TITLE
Build fail - Command "npm run vercel-build" exited

### DISCUSSION THREAD

**Discussion by snaps-david** [vercel/vercel#7022] [Help]
Hi!

I am new to Vercel and trying to deploy my project using it, but keep getting a `build fail` error.

I set the `builds` and `scripts` in `vercel.json` and `package.json` as instructed on the Documentation.

```
"builds": [
    {
      "src": "dist/server.js",
      "use": "@vercel/node",
      "config": { "includeFiles": ["dist/**"] }
    }
  ],
```

```
"scripts": {
    "vercel-build": "yum install libuuid-devel libmount-devel zlib && cp /lib64/{libuuid,libmount,libblkid,libz}.so.1 node_modules/canvas/build/Release/",
    "start": "webpack --config webpack.config.dev.js",
    "start:dev": "webpack --config webpack.config.dev.js && node ./dist/server.js",
    "build:dev": "webpack --config webpack.config.dev.js",
}
```

Does anyone know how I can fix this? Thanks!

**Comment by eofoneta** [comment]
hey I know its been long solved but i'm having same issue, how do i solve this?

**Reply by snaps-david** [reply]
If I remember correctly, I was unable to successfully install some libraries I needed using yum install.
I move to AWS EC2 instead.

**Reply by eofoneta** [reply]
vercel can be so tiring to deploy applications, i tried adding a few vercel.json to my file and it successfully deployed but i'm getting a 404 error from the website. so so exhausting
